src/Symfony/Component/Validator/ConstraintViolation.php

```diff
@@ -32,7 +34,7 @@ class ConstraintViolation implements ConstraintViolationInterface
     /**
      * Creates a new constraint violation.
      *
-     * @param string          $message         The violation message
+     * @param mixed           $message         The violation message
```

Comment:
let's keep it as `string` but alter the description instead, eg
`The violation message as a string or a stringable object`